packages/ckeditor5-ui/src/bindings/addkeyboardhandlingforgrid.ts

```diff
@@ -84,4 +82,52 @@ export default function addKeyboardHandlingForGrid(
 			evt.preventDefault();
 		};
 	}
+
+	// Function returning a right index relatively current index.
+	//
+	// before: [ ][x][ ]	after: [ ][ ][x]
+	//         [ ][ ][ ]	       [ ][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 1            index = 2
+	//
+	// If current index is at the right conner, function return first index of next row.
+	//
+	// before: [ ][ ][x]	after: [ ][ ][ ]
+	//         [ ][ ][ ]	       [x][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 2            index = 3
+	//
+	// @param {number} [elementIndex] Number of current index.
+	// @param {module:ui/viewcollection~ViewCollection} [gridItems] A collection of grid items.
+	function getRightElementIndex( elementIndex: number, gridItems: ViewCollection ) {
+		if ( elementIndex === gridItems.length - 1 ) {
+			return 0;
+		} else {
+			return elementIndex + 1;
+		}
+	}
+
+	// Function returning a left index relatively current index.
+	//
+	// before: [ ][x][ ]	after: [x][ ][ ]
+	//         [ ][ ][ ]	       [ ][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 1            index = 0
+	//
+	// If current index is at the left conner, function return last index of previous row.
+	//
+	// before: [ ][ ][ ]	after: [ ][ ][x]
+	//         [x][ ][ ]	       [ ][ ][ ]
+	//         [ ]      	       [ ]
+	// index = 2            index = 0
+	//
+	// @param {number} [elementIndex] Number of current index.
+	// @param {module:ui/viewcollection~ViewCollection} [gridItems] A collection of grid items.
+	function getLeftElementIndex( elementIndex: number, gridItems: ViewCollection ) {
```

Comment:
Same comments as for the previous function, only with a change in the docs - if the first index is passed, it returns the last one.